src/m365/spo/commands/page/page-template-list.ts

```diff
@@ -1,10 +1,10 @@
 import { Logger } from '../../../../cli/Logger';
 import GlobalOptions from '../../../../GlobalOptions';
-import request from '../../../../request';
+import { odata } from '../../../../utils/odata';
 import { validation } from '../../../../utils/validation';
 import SpoCommand from '../../../base/SpoCommand';
 import commands from '../../commands';
-import { PageTemplateResponse } from './PageTemplateResponse';
```

Comment:
Same here for `PageTemplateResponse`. Do we still have need for this interface? I think this was the last location it was used.

Reply:
Correct!